### PR TITLE
feat(dataset): add tornadoes-2024-dow-zdr-greenfield dataset config

### PR DESCRIPTION
**Pull request by anayeaye**
## What
Add stac collection and ingestion config for `tornadoes-2024-dow-zdr-greenfield`

## Parent issue
https://github.com/NASA-IMPACT/veda-data/issues/187

## Downstream veda-config PR
https://github.com/NASA-IMPACT/veda-config/pull/420

## Checklist
- [x] staging config completed and verified
- [x] published to staging
- [x] transfer config and production metadata updated
- [x] published to production

### CODE REVIEW

File: ingestion-data/production/dataset-config/tornadoes-2024-dow-zdr-greenfield.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
+    "collection": "tornadoes-2024-dow-zdr-greenfield",
+    "title": "DOW7 Differential Reflectivity (Greenfield, IA)",
+    "description": "Differential Reflectivity (dB) collected by DOW7 from the Greenfield, Iowa EF-4 tornado on May 21, 2024 at 20:41:08.",
```
Comment (botanical) [inline ingestion-data/production/dataset-config/tornadoes-2024-dow-zdr-greenfield.json:4]:
```suggestion
    "description": "Differential Reflectivity values (dB) from the EF-4 Greenfield, IA tornado on May 21, 2024 collected by DOW7. Scan taken at 20:41:08.",
```

File: ingestion-data/staging/dataset-config/tornadoes-2024-dow-zdr-greenfield.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
+{
+    "collection": "tornadoes-2024-dow-zdr-greenfield",
+    "title": "DOW7 Differential Reflectivity (Greenfield, IA)",
+    "description": "Differential Reflectivity (dB) collected by DOW7 from the Greenfield, Iowa EF-4 tornado on May 21, 2024 at 20:41:08.",
```
Comment (botanical) [inline ingestion-data/staging/dataset-config/tornadoes-2024-dow-zdr-greenfield.json:4]:
```suggestion
    "description": "Differential Reflectivity values (dB) from the EF-4 Greenfield, IA tornado on May 21, 2024 collected by DOW7. Scan taken at 20:41:08.",
```

**Review by botanical — APPROVED** [review]
Description difference but the collection itself in production looks good

**Comment by anayeaye** [comment]
@botanical I published the corrected descriptions to prod

**Pull request merged**